copy-json-to-bipf.js

```diff
@@ -11,15 +11,13 @@ var block = 64*1024
 if(process.argv[2] === process.argv[3]) throw new Error('input must !== output')
 
 var log = FlumeLog(process.argv[2], {blockSize: block, codec: json})
+var log2 = FlumeLogAligned (process.argv[3], {block: block})
 
-var log2 = FlumeLogRaf (process.argv[3], {block: block})
-var a = [], length = 0
 pull(
   log.stream({seqs:false, codec: json}),
   pull.map(function (data) {
     var len = binary.encodingLength(data)
     var b = Buffer.alloc(len)
-    length += b.length + 4
```

Comment:
This variable `length` (and `a`) were written to, but never **read**, so I figured we could delete it